team_access_integration_test.go

```diff
@@ -338,3 +338,68 @@ func TestTeamAccessesRemove(t *testing.T) {
 		assert.Equal(t, err, ErrInvalidAccessTeamID)
 	})
 }
+
+func TestTeamAccessesReadRunTasks(t *testing.T) {
+	skipIfFreeOnly(t)
+	skipIfBeta(t)
+	skipIfEnterprise(t)
+
+	client := testClient(t)
+	ctx := context.Background()
+
+	orgTest, orgTestCleanup := createOrganization(t, client)
+	defer orgTestCleanup()
+	wTest, wTestCleanup := createWorkspace(t, client, orgTest)
+	defer wTestCleanup()
+	tmTest, tmTestCleanup := createTeam(t, client, orgTest)
+	defer tmTestCleanup()
+
+	taTest, taTestCleanup := createTeamAccess(t, client, tmTest, wTest, orgTest)
+	defer taTestCleanup()
+
+	t.Run("when the team access exists", func(t *testing.T) {
+		ta, err := client.TeamAccess.Read(ctx, taTest.ID)
+		require.NoError(t, err)
+
+		assert.Equal(t, AccessAdmin, ta.Access)
+
+		t.Run("permission attributes are decoded", func(t *testing.T) {
+			assert.Equal(t, true, ta.RunTasks)
+		})
+	})
+}
+
+func TestTeamAccessesUpdateRunTasks(t *testing.T) {
+	skipIfFreeOnly(t)
+	skipIfBeta(t)
+	skipIfEnterprise(t)
```

Comment:
QQ: Is there a particular reason these tests should only run against TFE installations and not TFC?

Reply:
Ahh...huh.. the name of that method feels misleading.  Should be TFC only right now.

Reply:
Yeah 😞  this has been a source of confusion even for myself. We may need to revisit the naming of these helpers so that it's a lot clearer on when to use them.